README.md

```diff
@@ -1830,6 +1831,7 @@ Options:
 -   `port` Port that Redis should be available on
 -   `parameter_group` The set of Redis parameters to use
 -   `num_nodes` Number of nodes in cache cluster
+-   `maintenance_window` Specifies the weekly time range during which maintenance on the cache cluster is performed. 
```

Comment:
I added the default value as sunday from 1am to 2am. Would that be alright ?

I've also noted that the window needs to be at least an hour.

Thank you